plugins/MauticCrmBundle/Integration/DynamicsIntegration.php

```diff
@@ -479,6 +481,21 @@ public function getLeads($params = [], $query = null, &$executed = null, $result
         return $executed;
     }
 
+    /**
+     * Convert to UTC date string for API call.
+     *
+     * @param string $paramDate
+     *
+     * @return string
+     */
+    public function getFilterDateUTC($paramDate)
```

Comment:
Looking good. Let's use the PHP 7.4 types and save some space :)
```suggestion
    /**
     * Convert to UTC date string for API call.
     */
    public function getFilterDateUTC(string $paramDate): string
```